server/server.js

```diff
@@ -71,6 +71,29 @@ app.get("/api/v1/creature_types/:type", (req, res) => {
     });
 });
 
+<<<<<<< HEAD
```

Comment:
You have some merge markers left in this file. 
`<<<<<<< HEAD`
`=======`
`>>>>>>> 8287e82b46c4978dcf79a906826681c84b87e706`